pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "pip", "setuptools>=77" ]
```

Comment:
note: I've used `setuptools>=77` here to get support for `PEP 639` license specification (`project.license-files`), but unfortunately this version of setuptools isn't available for Python 3.8. Would it be okay to also drop support for this end-of-life version of Python too ?

Reply:
I'm in support of dropping it but Valentin should make the final call